shared/src/packages/shifts/libs/types/shift-entity.type.ts

```diff
@@ -0,0 +1,13 @@
+import { type UserEntityT } from '~/packages/users/users.js';
+
+import { type TruckEntity } from '../../../../packages/trucks/trucks.js';
+
+type ShiftEntity = {
```

Comment:
maybe change naming, entities are mostly in be folder
better type ShiftResponseDto, and then you don't need to create a type ShiftCreateResponseDto

Reply:
if you allow, I'll stay this type, it is convenient for understanding of entity and request-response structure for me. I've only renamed ShiftCreateResponseDto to ShiftResponseDto.